src/components/dashboard/main-content/course-enrollments/course-cards/styles/_CourseCard.scss

```diff
@@ -44,4 +45,10 @@
       display: block;
     }
   }
+
+  .badge {
```

Comment:
You should be using https://paragon-openedx.netlify.app/components/badge/ instead of applying custom CSS. Also, the badge background color is different in the mockups.